src/imaginghelper.js

```diff
@@ -689,8 +689,17 @@
      **/
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+            contentSize;
+        // check if world is defined
+        if (this._viewer.viewport.contentSize == undefined){
```

Comment:
This should be a `===`; otherwise, it could be any falsy value (which I suppose could be fine in this case, but then we should just test `!this._viewer.viewport.contentSize` if that's what we want).